docs/hello_nextflow/02_hello_world.md

```diff
@@ -841,44 +1041,101 @@ params.greeting = "Bonjour le monde!"
 
 _After:_
 
-```groovy title="hello-world.nf" linenums="38"
+```groovy title="hello-world.nf" linenums="6"
 /*
  * Pipeline parameters
  */
-params.input_file = "data/greetings.txt"
+params.input_file = "data/greetings.csv"
+```
+
+### 9.2. Update the channel declaration to handle the input file
+
+At this point we introduce a new channel constructor, `Channel.fromPath()`, which has some built-in functionality for handling file paths.
+We're going to use that instead of the `Channel.of()` constructor we used previously; the base syntax looks like this:
+
+```groovy title="channel construction syntax"
+Channel.fromPath(input_file)
+```
+
+Since our goal is to read in the contents of a `.csv` file, we're going to add the `.splitCsv()` operator to make Nextflow parse the file contents accordingly, as well as the `.flatten()` operator to turn the array element produced by `.splitCsv()` into a channel of individual elements.
```

Comment:
```suggestion
Now, we are going to deploy a new concept, an 'operator' to transform that CSV file into channel content. You'll learn more about operators later, but for now just understand them as ways of transforming channels in a variety of ways. 

Since our goal is to read in the contents of a `.csv` file, we're going to add the `.splitCsv()` operator to make Nextflow parse the file contents accordingly, as well as the `.flatten()` operator to turn the array element produced by `.splitCsv()` into a channel of individual elements.
```

I felt the need to actually introduce operators, since this is the first time we've mentioned them explicitly